src/pages/posts/[slug]/preview.tsx

```
@@ -41,17 +42,11 @@ export default function PostPreviewPage({ post }: PostPreviewPageProps) {
             <h1>{post.title}</h1>
             <time>{post.updatedAt}</time>
           </header>
-          <main
-            className={classes.preview}
-            dangerouslySetInnerHTML={{ __html: post.content }}
-          />
+
+          <main className={classes.preview} dangerouslySetInnerHTML={{ __html: post.content }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)